lego/apps/achievements/verification.py

```diff
@@ -32,6 +34,29 @@ def check_poll_responses(user: User, count: int):
     )
 
 
+def check_longest_period_without_penalties(user: User, years: int) -> bool:
+    days = years * 365
+
+    if not (events := Registration.objects.filter(user=user).order_by("end_time")):
```

Comment:
`if not (events := Registration.objects.filter(user=user).order_by("event__end_time")):`

Reply:
Is it the same for the next one, Penalty...?

Reply:
No, all models that inherit from "BasisModel" will have created_at (and updated_at) attached to it.